server/routes/termdb.topVariablyExpressedGenes.ts

```diff
@@ -208,17 +208,29 @@ function gdcValidateQuery(ds: any, genome: any) {
 		const url = path.join(host.geneExp, '/gene_expression/gene_selection')
 		try {
 			// cachedFetch will only cache a response if an external API URL is enabled in serverconfig.features.extApiCache
-			const response = await cachedFetch(url, {
-				method: 'POST',
-				headers,
-				body: getGeneSelectionArg(q)
-			})
+			const response = await cachedFetch(
+				url,
+				{
+					method: 'POST',
+					headers,
+					body: getGeneSelectionArg(q)
+				},
+				{
+					getErrMessage: r => {
+						const body = r.body || r
```

Comment:
why is `r.body` optional?

Reply:
Fixed, the argument is always response `body` for APIs that do not follow standard HTTP response codes.